Vagrantfile

```diff
@@ -98,6 +98,17 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     backend.vm.provision :shell, :inline => "puppet apply /vagrant/tests/sensu-backend-federated-cluster.pp"
   end
 
+  # Unable to use at this time
+  # https://github.com/dotless-de/vagrant-vbguest/issues/367
+  #config.vm.define "el8-agent", autostart: true do |agent|
+  #  agent.vm.box = "centos/8"
```

Comment:
from the issue, it seems like we can point to the URL for centos 8.1 and once centos does another release, this should hopefully be fixed and can be put back to `centos/8`.

Reply:
The issue is with vagrant-vguest, not centos boxes. The issue I linked is the issue in vbguest plugin that prevents virtualbox guest additions from being installed into CentOS 8 boxes. I don't think there is anything CentOS can do to fix this.

Reply:
I used work around of forcing CentOS 8.1 box and that solved the issues.